node:events:485
      throw er; // Unhandled 'error' event
      ^

Error: listen EADDRINUSE: address already in use :::8080
    at Server.setupListenHandle [as _listen2] (node:net:1939:16)
    at listenInCluster (node:net:1996:12)
    at Server.listen (node:net:2101:7)
    at Server.attach (/Users/vortex/Desktop/games/whot/whot-server/node_modules/socket.io/dist/index.js:176:17)
    at new Server (/Users/vortex/Desktop/games/whot/whot-server/node_modules/socket.io/dist/index.js:71:18)
    at module.exports (/Users/vortex/Desktop/games/whot/whot-server/node_modules/socket.io/dist/index.js:595:33)
    at Object.<anonymous> (/Users/vortex/Desktop/games/whot/whot-server/index.js:6:32)
    at Module._compile (node:internal/modules/cjs/loader:1734:14)
    at Object..js (node:internal/modules/cjs/loader:1899:10)
    at Module.load (node:internal/modules/cjs/loader:1469:32)
Emitted 'error' event on Server instance at:
    at emitErrorNT (node:net:1975:8)
    at process.processTicksAndRejections (node:internal/process/task_queues:90:21) {
  code: 'EADDRINUSE',
  errno: -48,
  syscall: 'listen',
  address: '::',
  port: 8080
}

Node.js v23.11.0
